test-end-to-end/android-maestro/common/add_new_student_inside_course.yaml

```diff
@@ -0,0 +1,30 @@
+appId: com.toughra.ustadmobile
+---
+- tapOn: "Courses"
+- tapOn:
+      id: "com.toughra.ustadmobile:id/item_clazzlist_clazz_cl"
+- tapOn: "MEMBERS"
+- tapOn: "Add a student"
+- tapOn: "Add a new person"
+- tapOn: "First names"
+- inputText: ${STUDENTFIRSTNAME}
+- tapOn: "Last name"
+- inputText: ${STUDENTLASTNAME}
+- tapOn: "Gender"
+- tapOn: ${GENDER}
+- tapOn: "SAVE"
+- tapOn: "SAVE"
+- swipe:
```

Comment:
What is this swipe for? Can you add a comment to the source? Are we sure those exact numbers will always work?

Reply:
There is a known issue with the maestro. Scroll functionality is not working for the emulator that we are using in Jenkins. So we are using the swipe command for the test. It's working as expected in this test.

Reply:
I see the comment added for this, thanks. Got it.